assets/js/modules/analytics/setup.js

```diff
@@ -368,21 +331,37 @@ class AnalyticsSetup extends Component {
 				} );
 			}
 
-			return new Promise( ( resolve ) => {
-				resolve( newState );
-			} );
+			// If tag hasn't been detected in wp_head, look for existing tag in the html.
+			if ( ! newState.existingTag && ! newState.errorCode ) {
+				const existingTag = await getExistingTag( 'analytics' );
```

Comment:
There is a problem with this logic (although it's not responsibility of this PR to fix it): The existing tag looked up via the server and via remote request should be handled similarly, so this whole clause here should probably happen right after checking whether `response` contained an existing tag, essentially as fallback.

The actual problem this currently presents is the following: We only filter the accounts and properties to show the right one selected in the dropdown for the tag detected from the server - but we don't do this if the tag was found via remote request. This causes the wrong account/property to be highlighted in the disabled dropdown. Not super-critical because the functionality in the end is unaffected, but it is confusing.

@lucymtc This should be easy enough to fix as part of #83, please keep an eye out. I added it as a requirement to the ACs on that issue:

> The correct account and property are preselected.